cypress/utils/dashboards/workspace-plugin/commands.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BASE_PATH } from '../../base_constants';
+import { WORKSPACE_API_PREFIX } from './constants';
+
+Cypress.Commands.add('deleteWorkspace', (workspaceName) => {
```

Comment:
How about renaming to `deleteWorkspaceByName`? Maybe we will have `deleteWorkspaceById` in the future.

Reply:
I've changed that, and also added a new command `deleteWorkspaceById `.